openapi/connector_mgmt-private-admin.yaml

```diff
@@ -1055,6 +1114,21 @@ components:
               type: string
               #format: date-time
 
+    ConnectorRequest:
```

Comment:
There is already a ConnectorRequest type defined in the referenced schema from `connector_mgmt.yaml`. Let's avoid overloading type names across the APIs. Can you think of another name, e.g. `ConnectorAdminRequest`?

Reply:
fixed in https://github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/pull/1291/commits/7ade896398fb40d4ffc9108ed504603badb87366